dom/events/Event-dispatch-detached-input-and-change.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<link rel="author" title="Joey Arhar" href="mailto:jarhar@chromium.org">
+<title>input and change events for detached checkbox and radio elements</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<body>
+<script>
+
+async_test(t => {
```

Comment:
It looks like these can all be normal sync `test()`s, with no need for `t` or `t.done()`.